src/main/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationRemindApplicant2Notification.java

```diff
@@ -41,15 +42,33 @@ public class ApplicationRemindApplicant2Notification implements ApplicantNotific
     public void sendToApplicant2(final CaseData caseData, final Long id) {
         log.info("Sending reminder to applicant 2 to review case : {}", id);
 
-        notificationService.sendEmail(
-            caseData.getApplicant2EmailAddress(),
-            JOINT_APPLICANT2_ANSWERS_SENT_FOR_REVIEW,
-            templateVars(caseData, id, caseData.getApplicant2(), caseData.getApplicant1(), true),
-            caseData.getApplicant1().getLanguagePreference()
-        );
+        if (caseData.getApplicant1().isRepresented()) {
+
+            notificationService.sendEmail(
+                caseData.getApplicant2EmailAddress(),
+                JOINT_APPLICANT2_ANSWERS_SENT_FOR_REVIEW_WHEN_APPLICANT1ISREPRESENTED,
```

Comment:
Could probably make this constant shorter and more succinct. Also if you are using snake_case, then would advise to maintain snake_case throughout the constant name